operator/roles/ansible-service-broker/templates/broker-config.configmap.yaml.j2

```diff
@@ -7,10 +7,37 @@ metadata:
   namespace: {{ broker_namespace }}
   labels:
     app: {{ broker_name }}
-{% if state == "present" %}
 data:
   broker-config: |
-{{ broker_config | to_nice_yaml | indent(4, true) }}
-{% else %}
-data: {}
+    registry: {{ registries | to_json }}
+    dao:
+      type: crd
+    log:
+      logfile: /var/log/ansible-service-broker/asb.log
+      stdout: true
+      level: "{{ log_level }}"
+      color: true
+    openshift:
+      host: ''
+      ca_file: ''
+      bearer_token_file: ''
+      image_pull_policy: "{{ apb_pull_policy }}"
+      sandbox_role: "{{ sandbox_role }}"
+      namespace: "{{ broker_namespace | quote }}"
+      keep_namespace: {{ keep_namespace }}
+      keep_namespace_on_error: {{ keep_namespace_on_error }}
+    broker:
+{% if dashboard_redirector_host is defined %}
+      dashboard_redirector: https://{{ dashboard_redirector_host }}
```

Comment:
If you want...you could simply put all of the logic for the dashboard redirector here:

```
{% if state == present %}
    dashboard_redirector: https://{{ lookup('k8s', api_version='route.openshift.io/v1', kind='Route', namespace=broker_namespace, resource_name=dashboard_redirector_route_name).spec.host }}
{% endif %}

Reply:
This is ultimately conditional based on whether or not routes are available, otherwise `dashboard_redirector_host` won't be defined.

Reply:
Yeah. I would like to believe that this line was blob was different when I made the comment but I can't be sure. Looks fine how it is now.